nilearn/plotting/displays.py

```diff
@@ -206,8 +206,9 @@ class GlassBrainAxes(BaseAxes):
     volumes with a schematic view of the brain.
 
     """
-    def __init__(self, ax, direction, coord, **kwargs):
+    def __init__(self, ax, direction, coord, plot_negative=False, **kwargs):
```

Comment:
I am wondering if we could find a better wording. Negative values are always plotted, they are just turned into positive values in one case. Maybe `plot_absolute=True`?